Technical modification/upgrades

GuillaumeDespaux opened the pull request:
Visual/Typos
- Fixing mermaid graph
- Removing duplicated anchor for the glossary
- Correction of typos
Coding
- Change the structure of Edges
- Change the structure  of graph

GuillaumeDespaux committed:
Terminal command modif. + New part

Adding the " at beginning of URL and at the end
Adding the part 13. Memory management to save memory

GuillaumeDespaux committed:
Reverse back the memory optimization + Typos

GuillaumeDespaux committed:
Test Mermaid fix

GuillaumeDespaux committed:
Removing anchor to be unique

GuillaumeDespaux committed:
Merge branch 'main' into technical

Petite-Brioche0 reviewed (approved):
everything seems correct

GuillaumeDespaux committed:
API MAJOR UPDATE

- Changing the templates by static option
- Format of the API route (the result format is now a parameter)
- Example of the route given, ?:landmark_1&:landmark_2&:format
